rpcs3/Emu/Cell/PPULLVMRecompilerCore.cpp

```diff
@@ -4560,10 +4562,10 @@ void Compiler::MFFS(u32 frd, u32 rc) {
 }
 
 void Compiler::MTFSF(u32 flm, u32 frb, u32 rc) {
-	u32 mask = 0;
+	u64 mask = 0;
```

Comment:
"	cr_i32 = m_ir_builder->CreateAnd(cr_i32, ~mask); // null ith nibble" takes a u64 as the second argument.
If mask is 32 bits then ~mask is 32 bits too and is zero extended ; the 32 upper bits of the second mask are 0ed.

This doesn't make a difference since cr_i32 is (hopefully) a 32 bits values but static analysis did complain about that.